components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -14869,8 +14870,6 @@ public EndpointRegistryEntry getEndpointRegistryEntryByUUID(String registryEntry
                     endpointRegistryEntry.setServiceType(rs.getString("SERVICE_TYPE"));
                     endpointRegistryEntry.setServiceURL(rs.getString("SERVICE_URL"));
                     endpointRegistryEntry.setMetaData(rs.getString("METADATA"));
-                    ResourceFile resourceFile = new ResourceFile(rs.getBinaryStream("ENDPOINT_DEFINITION"),"");
```

Comment:
why removed?

Reply:
It is added only here, the SQL query does not have `ENDPOINT_DEFINITION`. We are not retrieving the `definitionFile` in the SQL query - [1]. Do we need to return the definition file with the method `getRegistryEntryByUUID`?

[1] https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java#L1417